src/rgw/driver/d4n/rgw_sal_d4n.cc

```diff
@@ -2261,7 +2234,7 @@ int D4NFilterObject::D4NFilterDeleteOp::delete_obj(const DoutPrefixProvider* dpp
       ldpp_dout(dpp, 0) << "D4NFilterObject::" << __func__ << "(): Directory entry for: " << source->get_oid() << " blockid: " << fst << " block size: " << cur_len << " does not exist; continuing" << dendl;
       fst += cur_len;
       if (fst >= lst) {
-        break;
+	break;
```

Comment:
I'm not sure why this change is present. It seems like a formatting error.

Reply:
You must revert these changes if not made by you